python/pynessie/nessie_client.py

```diff
@@ -183,7 +183,14 @@ def cherry_pick(self: "NessieClient", branch: str, old_hash: Optional[str] = Non
         cherry_pick(self._base_url, branch, transplant_json, old_hash, self._ssl_verify)
 
     def get_log(
-        self: "NessieClient", start_ref: str, max_result_hint: Optional[int] = None, page_token: Optional[str] = None
+        self: "NessieClient",
+        start_ref: str,
+        max_result_hint: Optional[int] = None,
+        page_token: Optional[str] = None,
```

Comment:
Similiar to the above comment: maybe use "Python kwargs" here as well?

Reply:
fixed